mods/noita-mp/files/scripts/Ui.lua

```diff
@@ -334,7 +335,7 @@ function Ui.new()
         drawMenu()
         drawModConflictWarning()
 
-        if #EntityUtils.transformCache >= 10000 then
+        if EntityCache.size() >= 10000 then
```

Comment:
10000 was reasonable for cache in Lua. Need to verify manually while running Noita

Reply:
Used `EntityUtils.maxPoolSize`